cPP_DSC.adoc

```diff
@@ -237,7 +237,7 @@ Subjects: The following list contains the fundamental actors in the expected ope
 
 Objects: The following list contains objects the DSC expects to use during the expected operational use cases.
 
-* OB.P_SDO - Pre-provisioned SDOs (e.g. DSC.ID) with security attributes listed in the next paragraph.
+* OB.P_SDO - Pre-installed SDOs (e.g. DSC.ID) with security attributes listed in the next paragraph.
```

Comment:
This is one of the only two occurences of "pre-provisioned" in the document; everywhere else "pre-installed" is used. I suggest changing it here for consistency.